docs/index.rst

```diff
@@ -12,66 +12,58 @@ DepC Documentation
    :alt: Dependencies QOS
 
-Dependencies
-~~~~~~~~~~~~
-
-DepC allows you to push your nodes and their relationships into a Graph
-Database : we use `Neo4j <https://neo4j.com/>`__ as the Graph database
-to manage your nodes and their relationships.
-
-In order to scale, we created a Kafka consumer which listens some topics
-: you have just to send your nodes and their relationships using a
-:ref:`specific format <kafka>`, and we fully manage the database for you.
-We also provide a WebUI for a better user experience.
-
-.. note::
-   The nodes and their relationships are **versioned**,
-   so we can know exactly what are the dependencies of a node within a
-   particular period (which is a crucial need to compute an accurate QOS).
-
-QOS
-~~~
-
-A common usage in OVH is to create some probes that check a server
-health (for example), and send the result in a TimeSeries database. The
-data can be anything like the response time of a server, its RAM and CPU
-usage, its number of IO Waits, etc.
+DepC provides a CMDB component to store and request related nodes into a graph
+structure. These nodes can be anything : servers, products, customers or even
+web services. A typical use case would be to create some ``customers`` nodes,
+related to their ``products``. We can also link these ``products`` to
+``servers``. that way it would be very easy to display the impacted customers
+when a problem occurs on a server.
 
-Based on this principle, we have created a **Rule** system that queries
-the datapoints and computes a QOS from it. For example you can decide
-that any datapoints exceeding a given threshold will lower the QOS.
-These computing methods are explained in a :ref:`dedicated page <checks>`.
-
-.. figure:: _static/images/quickstart/rule_details_server_ping.png
-   :alt: QOS API
+Once DepC knows your dependency graph, it then becomes possible to compute a
+QoS for every nodes. By taking up our previous example, Depc can compute the
+QoS of our ``customers``, following the QoS of their ``servers``.
 
 .. note::
-   DepC currently supports 2 TimeSeries databases :
-   **OpenTSDB** & **Warp10**. Our goal is to add other data sources like
-   InfluxDB, Graphite or even Prometheus.
-
-Architecture
-------------
-
-.. figure:: _static/images/architecture.png
-   :alt: Architecture
-   :scale: 50 %
-   :align: center
+   QoS is not the same as SLA : please read :ref:`this guide <difference-qos-sla>`
+   to view the difference between them. Notions about SLO and Indicators are also
+   explained in it.
+
+Principles
+----------
+
+- **Graph Dependency** : We use the *Neo4j* database to manage the nodes and
+  their relationships (we already handle several million of nodes in our OVH
+  internal instance). DepC provides some API calls and WebUI to easily request
+  a node and its relations.
+- **QoS Computation** : DepC can compute a QoS for all nodes in the Graph, but
+  of course we must tell it how to do this. We created :ref:`different methods
```

Comment:
but of course we must tell it how to do this -> but of course we have to explain how we do it